packages/terraform-modules-public/aws/testnet/modules/proxy/main.tf

```diff
@@ -5,37 +5,54 @@ module "ami" {
 resource "aws_instance" "celo_proxy" {
```

Comment:
A cool idea. I was planning to solve this problem by backing up chaindata to s3 and providing an optional way to restore it. A data volume could also be a good option too. I'll probably start with the s3 option first (new PR) since that provides some more flexibility.

Reply:
Yeah, using S3 sounds great to me also for scaling perspective :+1: